site/docs/visual_elements/placeholders.mdx

```diff
@@ -0,0 +1,51 @@
+---
+name: Placeholders
+route: /visual-assets/placeholders
+menu: Visual assets
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Image from "../../src/components/Image";
+import Link from "../../src/components/Link";
+
+# Placeholders
+
+<LargeParagraph>Placeholders can be used to replace missing content or denote areas left intentionally blank.</LargeParagraph>
+
+Examples of possible use cases for placeholders:
+- In place of a missing image for example in a search listing blog posts, news items or events.
+- As a default image preview for an user uploaded image.
+
+## Usage
+Default HDS placeholders include **image placeholders** and **user image placeholders** for different sizes.
+
+### Image placeholders
+Different sizes of image placeholders available in HDS. 
+
+You can [download the image placeholder package](../../static/assets/placeholders/HDS-placeholders-image.zip "Download the image placeholder package zip file"), or just specific image files by opening images in the table bellow.
+
+Image                                                                                                                                                               | Size                  | Dimensions    | Pixel size      |
+------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------- | ------------- | --------------- |
+<Image src="../../static/assets/placeholders/image/image-s.png" alt="Placeholder image Small" style="width:100%; max-width: 150px;" viewable />                     | **Small**             | 150 x 100 dp  | 150 x 100 px    |
```

Comment:
Tämä "Small" kuva näyttää taulukossa jotenkin suttuiselta. Mietin, että johtuisi varmaan siitä miten tuo taulukko skaalaa kaikki kuvat saman kokoisiksi. Mutta vaikka tuon kuvan avaa, niin se on siinäkin suttuinen. Kannattaako meidän tarjota näin pientä kuvaa edes?

Reply:
No tätä mä mietin myös. Mullakin se näyttää suttuiselta, vaikka ikoni on aseteltu kuvaan pixel perfect. Johtuuko se vaan Macbookin näytön korkeammasta resoluutiosta? Kysymys kuuluukin oikeastaan: tarvitaanko @1x kuvia mihinkään? Pitäiskö ennemminkin tarjota @2x ja @3x?

Näissä eri kuvakoissahan ero on lähinnä niissä käytetyn ikonin koko. Small-koossa ikoni on 24px (tai dp, miten sen haluaa sanoa). Small-kokoa voi käyttää esim. thumbnaileissa. Mutta kuvan ulkomittojen olla suuremmat, että siitä on todellista hyötyä?

Reply:
Lisäsin nyt @3x koot ja tarkensin vähän ohjeistusta. Kaipa ne @1x kootkin on siellä tarjottava, vaikka niille ei käyttöä juuri oiskaan.